src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -146,10 +152,10 @@ const NavSearchBar = () => {
     [searchQuery]
   );
 
+  const inputRef = useRef<HTMLInputElement>(null);
   const focusSearchBar = useCallback((e) => {
     e.preventDefault();
-    const searchBar = document.getElementById("NavSearchBar__search-input");
-    if (searchBar) searchBar.focus();
+    if (inputRef && inputRef.current) inputRef.current.focus();
```

Comment:
F.ck, I could have sworn I tried this... May be without right for `<HTMLInputElement>`. Heh. Thank you!